component.yaml

```diff
@@ -0,0 +1,18 @@
+platform: iOS
+buildPlatform:
+displayName: Tuist Commands
+description: "Run Tuist Commands."
+inputs:
+- key: "AC_TUIST_PATH"
```

Comment:
IMO there should be two fields below as input for React and Flutter project to work:

title: Repository Directory
defaultValue: "$AC_REPOSITORY_DIR"

title: Project Path
defaultValue: "$AC_PROJECT_DIR"

Reply:
In the yaml file, the default value of AC_TUIST_PATH is already AC_REPOSITORY_DIR. In the source code, there is the -p parameter in the command line. This means it will look for the tuist tool in all subdirectories in AC_REPOSITORY_DIR. 

On the other hand, Tuist does not officially support react native and flutter platforms. But theoretically it could also work on cross platforms. I think we should consult @csonuryilmaz  about this.